syncano/models/fields.py

```diff
@@ -704,18 +742,51 @@ def validate(self, value, model_instance):
             except (ValueError, TypeError):
                 raise SyncanoValueError('Expected an object')
 
-        if not isinstance(value, GeoPointStruct):
-            raise SyncanoValueError('Expected an GeoPointStruct')
+        if not isinstance(value, GeoPoint):
```

Comment:
Using asserts in code that is meant to be normally used (outside of tests) is baaad. Especially for validations. 
`python -O` and suddenly my validations do not work ;)